config/default.yml

```diff
@@ -1446,23 +1446,31 @@ Lint/Debugger:
   Description: 'Check for debugger calls.'
   Enabled: true
   VersionAdded: '0.14'
-  VersionChanged: '0.49'
-  DebuggerReceivers:
-    - binding
-    - Kernel
-    - Pry
+  VersionChanged: <<next>>
+  DebuggerReceivers: [] # deprecated
```

Comment:
I didn't want users to get error messages about an invalid configuration (in case they've set this) so I've added it to the obsoletion config (with warning severity) but kept the configuration.